app/controllers/api/v2/sections_controller.rb

```diff
@@ -28,6 +28,26 @@ def show
         options[:include] = [:chapters, 'chapters.guides']
         render json: Api::V2::Sections::SectionSerializer.new(@section, options).serializable_hash
       end
+
+      def chapters
```

Comment:
Would it be better to make this the index of a chapters controller?

Reply:
the ChaptersController#index is already used 🤔 
maybe passing the section id too, 
we should use 2 different routes for the same controller's action,
I guess it would complicate the action and tests quite a bit.

Reply:
So I think we need Sections::ChaptersController#index, basically, to be resourceful and to match the route /sections/1/chapters